src/Integration.Vsix/BoundSolutionAnalyzer.cs

```diff
@@ -1,116 +0,0 @@
-﻿/*
```

Comment:
The class only existed to send telemetry data whenever a solution was opened. It's no longer required.